--- geant4.10.07.p04/source/processes/electromagnetic/dna/management/include/G4ITMultiNavigator.hh	2023-05-24 13:43:47.960214329 -0500
+++ geant4.10.07.p04/source/processes/electromagnetic/dna/management/include/G4ITMultiNavigator.hh	2023-05-24 14:13:50.595633742 -0500
@@ -82,11 +82,11 @@
   class G4TrackState<G4ITMultiNavigator> : public G4TrackState<G4ITNavigator>
   {
   public:
-    ~G4TrackState<G4ITMultiNavigator>()
+    ~G4TrackState()
     {
     }
 
-    G4TrackState<G4ITMultiNavigator>()
+    G4TrackState()
     {
       G4ThreeVector Big3Vector(kInfinity, kInfinity, kInfinity);
       fLastLocatedPosition = Big3Vector;
--- geant4.10.07.p04/source/processes/electromagnetic/dna/management/include/G4ITPathFinder.hh	2023-05-24 14:42:58.550496706 -0500
+++ geant4.10.07.p04/source/processes/electromagnetic/dna/management/include/G4ITPathFinder.hh	2023-05-24 14:43:17.886350945 -0500
@@ -116,9 +116,9 @@
 	G4int         fLastStepNo, fCurrentStepNo;
 
 public:
-	virtual ~G4TrackState<G4ITPathFinder>(){}
+	virtual ~G4TrackState(){}
 
-	G4TrackState<G4ITPathFinder>() :
+	G4TrackState() :
       G4TrackStateBase<G4ITPathFinder>(),
 			fEndState( G4ThreeVector(), G4ThreeVector(), 0., 0., 0., 0., 0.),
 			fFieldExertedForce(false),
